### PR TITLE
Install in CI

### PR DESCRIPTION
**Pull request by Dale-Koenig**

### CODE REVIEW

**Review by christianhelm — APPROVED** [review]

File: alica_dummy_proxy/CMakeLists.txt [hunk 1]
```diff
@@ -34,8 +34,8 @@ if (NOT catkin_FOUND)
  target_include_directories(alica_dummy_proxy PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")
 endif (NOT catkin_FOUND)
 
-install(DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+install(DIRECTORY include/
+  DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}
```
Comment (david-simoes-93) [inline alica_dummy_proxy/CMakeLists.txt:38]: these 2 are equivalent, right? previously you installed `include/pkg/` into `global/pkg/`, and now you install `include/` into `global/`

http://docs.ros.org/en/jade/api/catkin/html/user_guide/variables.html
>  CATKIN_GLOBAL_INCLUDE_DESTINATION
> This is set to include. This destination should only be used when installing headers which are not in a subfolder which matches the project name. Else you should use the destination CATKIN_PACKAGE_INCLUDE_DESTINATION instead.

Why the change? Seems like docs recommend the previous approach

Comment (david-simoes-93) [reply alica_dummy_proxy/CMakeLists.txt:38]: Ah, because project is `alica_dummy_proxy` and subfolder is `communication`?

**Review by david-simoes-93 — APPROVED** [review]

**Review by abhi1287 — APPROVED** [review]
